src/Map.js

```diff
@@ -2,11 +2,11 @@
 
 require('./style.css');
 const objectAssign = require('object-assign');
-const {isHTMLElement} = require('./utils/dom');
+const dom = require('./utils/dom');
```

Comment:
If we transpile this should not be necessary, no ?

Reply:
it's just for code style consistency